dask_cugraph/tests/test_dask_cugraph.py

```diff
@@ -0,0 +1,61 @@
+import pytest
```

Comment:
Let's add 2 other tests :
1. csv input only one file (using dask-cudf reader)
2. csv input one file per GPU

Reply:
-> this can go in the next PR